models/models.comments.js

```diff
@@ -36,3 +36,12 @@ exports.checkUserExists = (username) => {
     } else return true;
   });
 };
+
+exports.removeCommentByCommentId = (comment_id) => {
+  return Promise.all([db.query('SELECT * FROM comments WHERE comment_id=$1', [comment_id]), comment_id]).then(([{ rows }, comment_id]) => {
+    if (rows.length === 0) {
+      return Promise.reject({ status: 404, msg: 'Comment not found' });
+    }
+    return db.query('DELETE FROM comments WHERE comment_id = $1', [comment_id]);
```

Comment:
I would suggest doing your delete first and then checking if that comment exist after to make sure it has been deleted